system/Validation/FormatRules.php

```diff
@@ -246,7 +246,7 @@ public function valid_base64(string $str = null): bool
 	 */
 	public function valid_json(string $str = null): bool
 	{
-		json_decode($str);
+		json_decode($str, false, 512, JSON_THROW_ON_ERROR);
 		return json_last_error() === JSON_ERROR_NONE;
```

Comment:
Should this be handled by a try-catch now?

Reply:
Yes, probably need to be handled in the `rector` itself when next is `json_last_error()` check, try with Exception with message `json_last_error_msg()`